packages/node_modules/pouchdb-core/src/adapter.js

```diff
@@ -579,6 +579,9 @@ AbstractPouchDB.prototype.get = adapterFun('get', function (id, opts, cb) {
       var conflicts = collectConflicts(metadata);
       if (conflicts.length) {
         doc._conflicts = conflicts;
+        if (!doc._conflicts.length) {
+          delete doc._conflicts;
+        }
       }
```

Comment:
This shouldnt be needed given the lines preceeding it, I also think we should use this pattern in the other areas

    var conflicts = collectConflicts(metadata);
    if (conflicts.length) { 
      doc._conflicts = conflicts;
    }

Feels a lot cleaner than setting `doc._conflicts` then deleting if empty